roles/satellite-clone/tasks/main.yml

```diff
@@ -115,6 +115,13 @@
     name: "{{ satellite_package }}"
     state: latest
 
+# Install postgres server
```

Comment:
nitpick - First part of comment is redundant since its the name of the task (I know we do it some other places, but trying to break that pattern :) )

Reply:
agreed - ansible is very self-documenting